Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN echo -e "\n\nOS=$(cat /var/version)\n---------------\n" && \
   ./autogen.sh --prefix=/usr && \
   make && make install
 
-ENTRYPOINT ["make"]
-CMD ["check", "distcheck"]
+ENTRYPOINT []
+CMD ["make", "check", "distcheck"]
```

Comment:
I wonder if we really want this `CMD` in the Docker file?

In my mind, we want a Docker file to have an image ready for testing. It is the CI (travis) which then runs testing commands in it.